### DISCUSSION TITLE
Error while plotting, please advice?

### DISCUSSION THREAD

**Discussion by tjquan91** [Chia-Network/chia-blockchain#1414] [Q&A]
I just started and it was going fine until 

Bucket 89 uniform sort. Ram: 4.440GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
Bucket 90 uniform sort. Ram: 4.440GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
Bucket 91 uniform sort. Ram: 4.440GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
Bucket 92 uniform sort. Ram: 4.440GiB, u_sort min: 0.563GiB, qs min: 0.281GiB. 
**Only read 0 of 262143 bytes at offset 786429 from "D:\\Chia Temp\\plot-k32-2021-03-21-17......rt_bucket_092.tmp"with length 301984776. Error 1. Retrying in five minutes.**

and its been stuck there for the past 3 hours while keep retrying

**Comment by zachlankton** [comment]
Looks like its having trouble reading the disk.

Please share more info about your system/setup: OS, CPU, Ram, Type of Disk, Disk Size. Current version of chia.
Also share the parameters you are plotting with. k size, ram allocated, threads, bitfield yes/no? etc..

**Reply by FunkyPDB** [reply]
This happened to me when I tried installing on my main hard drive which had <300gigs of space. I had read you only needed 100gigs but after realize the final plot file was 100gigs, not the initial plot creation. Hope this helps.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
